terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -993,10 +993,14 @@
     end
 
     context "when the source type is an HTTP archive" do
-      let(:source_string) { "https://example.com/archive.zip?ref=v1.0.0" }
+      %w[.zip .bz2 .tar.bz2 .tar.tbz2 .tbz2 .gz .tar.gz .tgz .xz .tar.xz .txz].each do |extension|
```

Comment:
I updated the test for recognizing an archive http source to include all the extensions terraform respects, not just .zip